maintainers-guide.md

```diff
@@ -0,0 +1,210 @@
+  require immediate action. High demand, planned in the roadmap for the current
+  quarter.
+* **P2** - Important feature or defect that is tied to roadmaps or upcoming
+  releases. Moderate live issue in a released Bazel version that is inconvenient
+  for a user that needs to be addressed in a upcoming release.
+* **P3** - Desirable enhancement or minor bug fix that will benefit a user. Not
+  prioritized into Bazel roadmaps or any imminent release.
+* **P4** - Low priority defect or feature request that is unlikely to get
+  closed. Can also be kept open for a potential re-prioritization if more users
+  are impacted.
+
+## Team labels
+
+| Label                | Description                                                           | Link                                                                           |
+|----------------------|-----------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| team-Android         | Issues for Android team                                               | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Android)         |
+| team-Apple           | Issues for Apple team (also MacOS, iOS etc)                           | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Apple)           |
+| team-Configurability | Issues for Configurability team                                       | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Configurability) |
+| team-EngProd         | Issues for EngProd team (also release, distribution)                  | [Open issues](https://github.com/bazelbuild/bazel/labels/team-EngProd)         |
+| team-Execution       | Issues for Execution phase team (local and remote execution, sandbox) | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Execution)       |
+| team-ExternalDeps    | Issues for External Deps team (remote repositories, WORKSPACE file)   | [Open issues](https://github.com/bazelbuild/bazel/labels/team-ExternalDeps)    |
+| team-Performance     | Issues for Performance teams                                          | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Performance)     |
+| team-Product         | Issues for the Product team                                           | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Product)         |
+| team-Rules-CPP       | Issues for C++ rules                                                  | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Rules-CPP)       |
+| team-Rules-Java      | Issues for Java rules                                                 | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Rules-Java)      |
+| team-Rules-Python    | Issues for the native Python rules for Python                         | [Open issues](https://github.com/bazelbuild/bazel/labels/team-Python)          |
```

Comment:
cc @brandjon 
Shall we use https://github.com/bazelbuild/rules_python instead?

Reply:
While I do want more Python related discussions to happen on rules_python, at the moment I think it's ok to continue using the bazel repo for issues with the native rules.

Reply:
Since this label is for the native python ruleset, we can keep it here. If we receive issues regarding the rules_python implementation, we can direct them there instead.